app/Models/EventSetting.php

```diff
@@ -45,7 +45,7 @@ class EventSetting extends Model
 
     protected $guarded = [];
 
-    public function event(){
+    public function event() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->